src/common/components/Inputs/Input.tsx

```diff
@@ -1,16 +1,14 @@
-import React, { ReactNode, ReactSVGElement } from 'react'
+import React, { ReactNode } from 'react'
 import cn from 'classnames'
-import Icon from '../Icon/Icon'
+import Icon from '../Icon'
 import CheckIcon from '../../assets/icons/ico-check.svg'
 import TimesIcon from '../../assets/icons/ico-times.svg'
-import NumberFormat from 'react-number-format'
 
 export type TInput = {
   className?: string
-  type?: 'text' | 'password' | 'number'
-  kind?: 'numberFormat'
-  prepend?: ReactSVGElement
-  append?: ReactSVGElement
+  type?: string
```

Comment:
string can be accept any value, by default it is "text" in this component, but I see that InputPassword component provides "string" as a type.

Please turn it to the union of acceptable values, so typescript could check for valid types